src/app/(main)/my-voyage/[teamId]/voyage-resources/components/ResourceCard.tsx

```diff
@@ -80,13 +80,14 @@ export default function ResourceCard({
             <Badge title={user.firstName} avatarUrlImage={user.avatar} />
           </div>
           <div className="h-5 w-1 border border-y-0 border-l-0 border-r-neutral-content"></div>
-          <div className="text-neutral">Added {date}</div>
+          <div className="text-neutral-focus">Added {date}</div>
```

Comment:
Is there an issue here with neutral color? In the figma, it is neutral color, not neutral focus

Reply:
there's not enough contrast between the text color and background color so thats why i used focus

Reply:
ok. We can't just randomly change the color like this. Will bring it up to design team and see if they want to go this route or a different one